src/bindings/python/src/compatibility/pyngraph/node.cpp

```diff
@@ -278,16 +278,6 @@ void regclass_pyngraph_Node(py::module m) {
                 get_rt_info : PyRTMap
                     A dictionary of user defined data.
              )");
-    node.def("get_version",
```

Comment:
this API is not deprecated in python
@ilyachur do we need to provide alternative solution for BW compatibility? E.g. parse `string` to `int` and return `int` as it was before

Reply:
We can return opset version